toolkit/views/character_generator/character_elements.py

```diff
@@ -85,6 +85,9 @@ def is_valid(self) -> bool:
         alignment_list = Character_Generator.ALIGNMENT_DICT.get("All")
         if (
             self.alignment.value != "All"

```

Comment:
all the `!=` can be shortened to `self.alignment.value not in Character_Generator.ALIGNMENT_DICT`